drivers/properties/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,10 @@
+properties:
+  vendor:
```

Comment:
I don't think vendor & model make sense here since we can extract them from devicetree.

Reply:
I added my opinion to this as part of previous comment :)